geometry/eolearn/geometry/transformations.py

```diff
@@ -200,19 +200,22 @@ def execute(self, eopatch):
         if eopatch.bbox is None:
             raise ValueError('EOPatch has to have a bounding box')
 
+        feature_type, feature_name = self.raster_feature
+        height, width = self._get_raster_shape(eopatch)
+
         rasterization_shapes = self._get_rasterization_shapes(eopatch)
         if not rasterization_shapes:
+            eopatch[feature_type][feature_name] = \
+                np.full((height, width), self.no_data_value)[..., np.newaxis].astype(self.raster_dtype)
```

Comment:
Nicer and quicker way: `np.full((height, width, 1), self.no_data_value, dtype=self.raster_dtype)`